frontend/src/app/app.module.ts

```diff
@@ -10,15 +10,15 @@ import { RedirectComponent } from './redirect.component';
 import { WindowService, WindowFactory } from './window.service';
 import { HomeComponent } from './home/home.component';
 import { FooterComponent } from './footer/footer.component';
-import { NavbarComponent } from './navbar/navbar.component';
+import { HeaderComponent } from './header/header.component';
 import { SubFooterComponent } from './sub-footer/sub-footer.component';
 
 
 const appRoutes: Routes = [
-  // {
-  //   path: '',
-  //   component: HomeComponent
-  // },
+  {
```

Comment:
don't leave it uncommented (as we are not ready to go with this angular component)